src/pages/home/ReportScreen.js

```diff
@@ -227,7 +227,10 @@ class ReportScreen extends React.Component {
             reportClosedAction = lodashFindLast(this.props.reportActions, action => action.actionName === CONST.REPORT.ACTIONS.TYPE.CLOSED);
         }
         return (
-            <ScreenWrapper style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}>
+            <ScreenWrapper
+                style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
+                keyboardAvoidingViewBehavior={Platform.OS === 'android' ? '' : 'padding'}
```

Comment:
We would refactor this to follow our JS guidelines

Reply:
Or maybe we just ... wont? IDK...

Reply:
I mean if we want we could create something like `getReportScreenScreenWrapperStyles()` -> `index.js` `'padding'` `index.android.js` `''`, but not really worth IMO.

Reply:
Coming in here because I ran into this warning:

![image](https://user-images.githubusercontent.com/1228807/191115024-6706927a-32ab-4f8b-8586-31f296b50a9b.png)

Question: Why doesn't this default to "height" for android since that's what's being done in the android file above?

Request: Please refactor this to follow our platform conventions. Leaving code like this only let's contributors see the pattern and follow it. I recently read through the [README](https://github.com/Expensify/App) section "Cross Platform 99.9999%" and it's a good refresher of why we do this. We should not compromise here.

Reply:
There was some conversation about this being temporary to help unblock the deploy but I'm guessing it just did not get cleaned up.

Reply:
IIRC @Julesssss [found a better solution](https://github.com/Expensify/App/pull/10648) for this problem.